master/usage/configuration/bgp.md

```diff
@@ -194,7 +194,7 @@ To remove a BGP peer that was configured with IP address aa:bb::ff (the AS
 number is not required), peering with {{site.prodname}} node "node1", run the following
 command from any node:
 
-	$ calicoctl delete bgpPeer aa:bb::ff --scope=node --node=node1
+	$ calicoctl delete bgpPeer aa:bb::ff --node=node1
```

Comment:
this should be `calicoctl delete bgppeer bgppeer-node-aabbff`, the `--node` flag doesn't exist anymore